dynamic-app/src/main/java/com/okta/idx/android/dynamic/auth/DynamicAuthViewModel.kt

```diff
@@ -126,19 +122,23 @@ internal class DynamicAuthViewModel(private val recoveryToken: String) : ViewMod
     }
 
     private suspend fun handleResponse(response: IdxResponse) {
+        // If a response is successful, immediately exchange it for a token and set it on LoggedInUserView
         if (response.isLoginSuccessful) {
             when (val exchangeCodesResult =
                 client?.exchangeInteractionCodeForTokens(response.remediations[IdxRemediation.Type.ISSUE]!!)) {
                 is IdxClientResult.Error -> {
                     _state.value = DynamicAuthState.Error("Failed to call resume")
                 }
                 is IdxClientResult.Success -> {
+                    cancelPolling()
```

Comment:
While this doesn't hurt, it's not really needed, it'll be cancelled automatically once we navigate away from this page.